src/dscanner/stats.d

```diff
@@ -30,25 +32,122 @@ pure nothrow bool isLineOfCode(IdType t)
 	}
 }
 
-ulong printTokenCount(Tokens)(File output, string fileName, ref Tokens tokens)
+pure nothrow bool __isLineOfCode(TOK t)
 {
+	switch (t)
+	{
+	case TOK.semicolon:
```

Comment:
It seems like your indentation level differs from the rest of the project. Please keep the same indentation as before

Reply:
Yes, please fix this indentation.

Reply:
Also, why the double underscore in the function name?